execution/execution.go

```diff
@@ -32,6 +32,9 @@ import (
 	"github.com/hyperledger/burrow/txs"
 )
 
+// TODO
+const GasLimit = uint64(1000000)
```

Comment:
we should be exposing this as a genesis config option. not now, but for later.

Reply:
yes, I was planning to address this and other things with the ValidatorTx changes